### ISSUE TITLE
Support customize optimizer types and learning rates of models.

### ISSUE THREAD

**Issue by Echo9573** [sql-machine-learning/sqlflow#1656]
It is recommended to support custom optimizer types and learning rates in the with of training sentences of sqlflow_models.

**Comment by typhoonzero** [comment]
In fact, we also need to specify loss using WITH clause.

**Comment by Echo9573** [comment]
@typhoonzero yes, you are right.
